scripts/ci/build/derive-service-state.py

```
@@ -131,8 +149,9 @@ def get_latest_service_tag(service: Service, tag_length: int = 8) -> str:
     """
 
     paths = get_paths(service)
+    exclude_paths = [f":(exclude,glob){p}" for p in get_exclude_paths(service)]
     latest_sha = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)